eng/pipelines/templates/steps/vmr-validate-installers.yml

```diff
@@ -0,0 +1,66 @@
+parameters:
+- name: continueOnError
+  type: boolean
+  default: false
+
+- name: targetArchitecture
+  type: string
+  default: ''
+
+- name: OS
+  type: string
+  values:
+    - Windows_NT
+    - Linux
+    - Darwin
+
+steps:
+- task: DownloadBuildArtifacts@1
+  inputs:
+    artifactName: BlobArtifacts
+    downloadPath: $(Build.ArtifactStagingDirectory)
+    checkDownloadedFiles: true
+  displayName: Download Blob Artifacts
+
+- task: DownloadBuildArtifacts@1
+  inputs:
+    artifactName: PackageArtifacts
+    downloadPath: $(Build.ArtifactStagingDirectory)
+    checkDownloadedFiles: true
+  displayName: Download Package Artifacts
+
+# This is necessary whenever we want to publish/restore to an AzDO private feed
+# Since sdk-task.ps1 tries to restore packages we need to do this authentication here
+# otherwise it'll complain about accessing a private feed.
+- task: NuGetAuthenticate@1
+  displayName: 'Authenticate to AzDO Feeds'
+
+- ${{ if eq(parameters.OS, 'Linux') }}:
+  - script: |
+      extraBuildProperties="/p:TestRpmPackages=true"
+      if [[ '${{ parameters.targetArchitecture }}' == 'x64' ]]; then
+        # At the moment Deb packages are only available for x64
+        extraBuildProperties="$extraBuildProperties /p:TestDebPackages=true"
+      fi
+
+      ./build.sh \
```

Comment:
I think you are missing the `--ci` flag here which is one of the more important args as it controls whether the machine wide package cache gets used or not (for non-source-only builds).

Reply:
Oh, yeah, I don't use it locally, but other official legs/steps do use it - will fix, thanks.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/47313/commits/5dfa459dd9c421c17e8dfb70f3a2d32910f57e5f